frontend/templates/troubleshooting/Resource.tsx

```diff
@@ -0,0 +1,85 @@
+import { Stack, Image, Text, Badge } from '@chakra-ui/react';
+import { Guide } from './hooks/GuideModel';
+import { FaIcon } from '@ifixit/icons';
+import { faGauge, faClock } from '@fortawesome/pro-solid-svg-icons';
+
+export function GuideResource({ guide }: { guide: Guide }) {
+   return <Resource title={guide.title} />;
+}
+
+function ResourceBox({ children }: React.PropsWithChildren<{}>) {
+   return (
+      <Stack
+         alignSelf="stretch"
+         justify="flex-start"
+         align="flex-start"
+         spacing="0px"
+         overflow="hidden"
+         borderColor="gray.400"
+         borderWidth="1px"
+         borderRadius="4px"
+         minHeight="88px"
```

Comment:
Oddly specific, but I trust you here

Reply:
I think this was pulled from the mockup.